core/settings/qPref.cpp

```diff
@@ -3,6 +3,8 @@
 #include "qPref.h"
 #include "ssrf-version.h"
 
+QSettings qPref_setting;
+
```

Comment:
the commit message does NOT describe what you are doing in this commit

Reply:
as far as I can see (maybe wrong) this is not part of the CloudStorage PR, but the included dependency (which will be removed once #1518 is on master). I believe this is lobg gone in #1518.